account/manager.go

```diff
@@ -194,9 +194,7 @@ func (m *Manager) start() error {
 		// TODO(guggero): Refactor this to extract the init/funding
 		// part so we properly abandon the account if it fails before
 		// publishing the TX instead of trying to re-fund on startup.
-		err := m.resumeAccount(
-			ctx, account, true, false, DefaultFundingConfTarget,
-		)
+		err := m.resumeAccount(ctx, account, true, false, 0)
```

Comment:
Should we add a comment here that the fee rate of 0 means "should estimate before publishing"?

Reply:
Or perhaps we should extract the call to `m.cfg.Wallet.EstimateFee()` to just before this call so we assume the fee rate is always non-zero in `resumeAccount()`?